### DISCUSSION TITLE
Is there a way to invite friends?

### DISCUSSION THREAD

**Discussion by WreckingBANG** [legendary-gl/legendary#455] [Q&A] [answered]
I am playing Fall Guys at the Moment and i was wondering how i can invite people now. Normally there would be a pop-up from the epic games launcher and then i would invite friends there. Is there a option to do this or is it planned?

**Comment by CommandMC** [comment, marked as answer]
Look into `legendary eos-overlay` and its sub-commands

On Windows, all you have to do is run `legendary eos-overlay install`. On Linux it's a little more involved, and it's not working at all on Mac as far as I know.

**Comment by WreckingBANG** [comment]
Thank you!
